src/controller/bankrupt/bankrupt.controller.ts

```diff
@@ -4,6 +4,7 @@ import { logger, formattingOfficersInfo, userSession } from '../../utils';
 import { fetchBankruptOfficers } from '../../service';
```

Comment:
Does this file actually need changing as the changes are superficial and don't seem related to adding the discharge date?